vnext/Microsoft.ReactNative.Managed.CodeGen/CodeAnalyzer.cs

```diff
@@ -813,7 +813,7 @@ private static void CollectTypes(INamedTypeSymbol type, ICollection<INamedTypeSy
             }
         }
 
-        private bool TryFindAttribute(ISymbol symbol, INamedTypeSymbol attributeType, out AttributeData attr)
+        private bool TryFindAttribute(ISymbol symbol, INamedTypeSymbol attributeType, out AttributeData? attr)
```

Comment:
>out [](http://example.com/codeflow?start=86&length=3)

It would be nice to add [NotNullWhen(returnValue: true)] here and then we do not need the null checks added in the code above.

Reply:
IS that a thing? The `bool TryGetX(out X? x)` pattern is one of my favorites and I'm always null checking.